sgt-back-end/sgt-back-end/index.js

```diff
@@ -0,0 +1,110 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+    from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.status(200).json(grades);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({ error: 'Unexpected error occured.' });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  if (req.body.name === undefined || req.body.course === undefined || Number.isNaN(req.body.score) || req.body.score < 0 || req.body.score > 100) {
+    res.status(400).json({ error: 'Please include name, course, and a score between 0-100' });
+    return;
+  }
```

Comment:
This would be better as 2 `if` statements so you can give more specific error messages. 1 for missing data, and 1 for score being invalid or out of range.